index.html

```diff
@@ -843,11 +846,30 @@
 			*****************************************************
 		*/
 
+		// generalised code to add district info
+		function expandDistrictInfo(district) {
+			// make sure we have a district to use
+			if (district.length > 0 && district[0].layer.id === 'school_house_senate_districts_UNION-poly') {
+				data = district[0].properties;
+				var html = "<hr />";
+				html += "<span class='varname'>";
+				html += showHouseDistricts ? "House District: " : "Senate District: ";
+				html += "</span> <span class='attribute'>";
+				html += showHouseDistricts ? data.HseDistNum : data.SenDistNum;
+				html += "</span>";
+				return html;
+			} else {
+				// if there's no appropriate district match, then just return an empty string so we don't get "undefined" in the popup
+				return '';
+			}
+		}
+
 		//raising blended learners campuses popup
 			map.on('click', 'raising-blended-learners-campuses-points', function (e) {
+				var district = map.queryRenderedFeatures(e.point, {layers: ['school_house_senate_districts_UNION-poly']});
```

Comment:
This checks specifically for which object in the invisible districts layer has been clicked on